### ISSUE TITLE
Convert install part into code block

### ISSUE THREAD

**Issue by sadikkuzu** [sadikkuzu/squiral#47]
before:
`pip install squiral`

after:
```shell
pip install squiral
```

**Comment by Ben-Ostrovsky** [comment]
Hey man! I would love to sort out this problem for you. Can you explain the exact issue?

**Comment by sadikkuzu** [comment]
Hey @Ben-Ostrovsky  👋🏼 
Thanks for your interest. In `README.md` file,

```markdown
` ` ` shell
pip install squiral
` ` `

instead of

`pip install squiral`
```
<img width="583" alt="image" src="https://user-images.githubusercontent.com/23168063/188336909-2c750af5-7516-4a41-91d4-f915ad819d03.png">

Result will be like that:

### Install

```shell
pip install squiral
```

<img width="154" alt="image" src="https://user-images.githubusercontent.com/23168063/188337102-b468534d-8f52-4407-b397-6ac06f96ebbb.png">

**Comment by sadikkuzu** [comment]
Resolved by #48
